### PR TITLE
Use gppApplicableSections and gppString In LiveIntent's Prebid User Id Module

### PR DESCRIPTION
**Pull request by wi101**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
LiveConnect added support for gpp.
In the LiveIntent UserId, forward the gppSections and gppString to set the LiveConnect gpp configuration.

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@wi101 can you take out the package-lock change? We are a few versions ahead of the change now

**Comment by wi101** [comment]
@ChrisHuie I updated the branch with master and ran `npm install` it fixed the version in master as well. Should I leave that change in package-lock  ?

**Comment by ChrisHuie** [comment]
> @ChrisHuie I updated the branch with master and ran `npm install` it fixed the version in master as well. Should I leave that change in package-lock ?

Yeah. This is fine :)

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "prebid.js",
-      "version": "8.21.0-pre",
+      "version": "8.24.0-pre",
```
Comment (patmmccann) [inline package-lock.json:9]: please remove this change

**Pull request merged**